jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -678,9 +687,14 @@ class DeploymentFlow extends React.Component {
           applications={this.props.applications}
           changeState={this.props.changeState}
           charms={this.props.charms}
+          estimate={this.state.ddEntity.toEntity().price}
```

Comment:
`estimate={this.state.ddEntity.get('price')}`?

Reply:
done